source/maintenance_chart_checker.py

```diff
@@ -0,0 +1,211 @@
+"""Maintenance Chart Checker
+
+This script analyzes maintenance charts in reStructuredText (.rst) files. It recursively scans
+through directories looking for .rst files containing maintenance charts in the format:
+
+**Maintenance chart**
```

Comment:
The Site Ops dir doesn't bold the maintenance chart title, and the Community pages have no maintenance charts at all. I'll fix this so we get a more accurate report.

Reply:
#1009